src/main/java/frc/robot/RobotContainer.java

```diff
@@ -55,26 +55,26 @@
 
   // subsystems
   private static DriveBaseSubsystem driveBaseSubsystem;
-  private static CDSSubsystem CDSSubsystem;
+  private static CDSSubsystem CDSSubsystem = new CDSSubsystem();
```

Comment:
Don't instantiate this here. Let this happen in the modular subsystem inside the RobotContainer's constructor

Reply:
I did